### PR TITLE
Port trivial cases

### LINKED ISSUE #134: Port trivial cases to new tutorial structure
Simply follow the guidelines defined [here](https://github.com/orgs/precice/projects/5)
And copy from how the `turek-hron-fsi3` case is done [here](https://github.com/precice/tutorials/tree/restructure/turek-hron-fsi3).

  * [ ] `flow-over-heated-plate-steady-state`
  * [x] `heat-exchanger`
  * [x] `elastic-tube-3d`
  * [x] `multiple-perpendicular-flaps`
  * [x] `partitioned-elastic-beam`

### PR DESCRIPTION
**Pull request by DavidSCN**
Closes #134 

 - `flow-over-heated-plate-steady-state`
    - [x]  README
    - [ ] test code_aster case (I have no installation)
 - `heat-exchanger`
    - [x]  README
 - `elastic-tube-3d`
    - [x]  README
-  `multiple-perpendicular-flaps`
    - [x]  README
- `partitioned-elastic-beam`
    - [x]  README

### CODE REVIEW

**Comment by uekerman** [comment]
> Although various cases were previously bad documented and I don't want to write the documentation for all of these cases.

Could you please split up the TODO above into one per case and let us know in which status each README is. We could then try to distribute the work there.

**Comment by DavidSCN** [comment]
@MakisH may I ask you to test the code_aster tutorial? I think it will currently not work since there are specific requirements for the exchange directory (potentially incompatible with other solvers which we currently don't have).

**Comment by MakisH** [comment]
I also don't have code_aster installed at the moment to test it, but I should eventually include it in the VM. Have you tested it in the meantime? I guess having someone else to also go through the docs would be useful: https://github.com/precice/code_aster-adapter/wiki/Flow-over-plate-Code_Aster-Tutorial

The main things that are special are:
- that the user needs to set an absolute path as an exchange directory (we already specify this in the docs)
- that the case is steady-state.

I don't think I will have time to look into code_aster until after the workshop.

**Comment by DavidSCN** [comment]
> I also don't have code_aster installed at the moment to test it, but I should eventually include it in the VM. Have you tested it in the meantime? I guess having someone else to also go through the docs would be useful: https://github.com/precice/code_aster-adapter/wiki/Flow-over-plate-Code_Aster-Tutorial

I have already ported the docs to the new website.

File: multiple-perpendicular-flaps/precice-config.xml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+    <mesh name="Fluid2-Mesh-Centers">
+      <use-data name="Stress2" />
+    </mesh>
+
+    <mesh name="Fluid2-Mesh-Nodes">
+      <use-data name="Displacement2" />
+    </mesh>
+
+    <mesh name="Solid1_mesh">
```
Comment (uekerman) [inline multiple-perpendicular-flaps/precice-config.xml:32]:
```suggestion
    <mesh name="Solid1-Mesh">
```
and similar occurences below.

File: multiple-perpendicular-flaps/precice-config.xml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        direction="read"
+        from="Solid1_mesh"
+        to="Fluid1-Mesh-Nodes"
+        constraint="consistent" />
+      <mapping:nearest-neighbor
+        direction="read"
+        from="Solid2_mesh"
+        to="Fluid2-Mesh-Nodes"
+        constraint="consistent" />
+      <export:vtk directory="precice-output" />
```
Comment (uekerman) [inline multiple-perpendicular-flaps/precice-config.xml:63]:
```suggestion
```
or do we need those?

File: multiple-perpendicular-flaps/precice-config.xml [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+      <use-mesh name="Fluid1-Mesh-Centers" from="Fluid" />
+      <read-data name="Stress1" mesh="Solid1_mesh" />
+      <write-data name="Displacement1" mesh="Solid1_mesh" />
+      <watch-point mesh="Solid1_mesh" name="flap_tip" coordinate="-1.05;1" />
+      <mapping:nearest-neighbor
+        direction="read"
+        from="Fluid1-Mesh-Centers"
+        to="Solid1_mesh"
+        constraint="consistent" />
+      <export:vtk directory="precice-output" />
```
Comment (uekerman) [inline multiple-perpendicular-flaps/precice-config.xml:77]:
```suggestion
```
or do we need those?

File: multiple-perpendicular-flaps/precice-config.xml [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+    </participant>
+
+    <participant name="Solid2">
+      <use-mesh name="Solid2_mesh" provide="yes" />
+      <use-mesh name="Fluid2-Mesh-Centers" from="Fluid" />
+      <read-data name="Stress2" mesh="Solid2_mesh" />
+      <write-data name="Displacement2" mesh="Solid2_mesh" />
+      <watch-point mesh="Solid2_mesh" name="flap_tip" coordinate="0.95;1" />
```
Comment (uekerman) [inline multiple-perpendicular-flaps/precice-config.xml:85]:
```suggestion
      <watch-point mesh="Solid2-Mesh" name="Flap-Tip" coordinate="0.95;1" />
```

File: multiple-perpendicular-flaps/precice-config.xml [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+      <mapping:nearest-neighbor
+        direction="read"
+        from="Fluid2-Mesh-Centers"
+        to="Solid2_mesh"
+        constraint="consistent" />
+      <export:vtk directory="precice-output" />
```
Comment (uekerman) [inline multiple-perpendicular-flaps/precice-config.xml:91]:
```suggestion
```
or do we need those?

File: multiple-perpendicular-flaps/precice-config.xml [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+    <coupling-scheme:multi>
+      <time-window-size value="0.01" />
+      <max-time value="5" />
+      <participant name="Fluid" control="yes" />
+      <participant name="Solid1" />
+      <participant name="Solid2" />
+      <exchange data="Stress1" mesh="Fluid1-Mesh-Centers" from="Fluid" to="Solid1" />
+      <exchange data="Stress2" mesh="Fluid2-Mesh-Centers" from="Fluid" to="Solid2" />
+      <exchange data="Displacement1" mesh="Solid1_mesh" from="Solid1" to="Fluid" initialize="0" />
+      <exchange data="Displacement2" mesh="Solid2_mesh" from="Solid2" to="Fluid" initialize="0" />
```
Comment (uekerman) [inline multiple-perpendicular-flaps/precice-config.xml:106]:
```suggestion
      <exchange data="Displacement1" mesh="Solid1_mesh" from="Solid1" to="Fluid" />
      <exchange data="Displacement2" mesh="Solid2_mesh" from="Solid2" to="Fluid" />
```

**Review by uekerman — CHANGES_REQUESTED** [review]
Some suggestion on "multiple flaps". Similarly, the preCICE config of "partitioned elastic beam" and "flow-over-heated-plate" needs to be ported. The latter, in particular, needs to be changed to 2D.

**Comment by uekerman** [comment]
> I also don't have code_aster installed at the moment to test it, but I should eventually include it in the VM. Have you tested it in the meantime? I guess having someone else to also go through the docs would be useful: https://github.com/precice/code_aster-adapter/wiki/Flow-over-plate-Code_Aster-Tutorial

I guess what @MakisH meant is to try them out :grin:

File: multiple-perpendicular-flaps/precice-config.xml (hunk 2 above)
Comment (DavidSCN) [reply multiple-perpendicular-flaps/precice-config.xml:63]: They are explicitly mentioned in the README. Let's keep them here.

Comment (uekerman) [reply multiple-perpendicular-flaps/precice-config.xml:63]: I removed them there as well :smile: 
In my opinion, preCICE vtk is for debugging, not to show the solid results. The interface displ you already get from OpenFOAM and for more information (stress distribution etc) you need the full solid results. This interface output, you really only need if you don't trust what preCICE is doing, so debugging, which should not be on by default :smirk:

**Comment by uekerman** [comment]
Let's ignore the Code_Aster case here and do it after merging: #155.

**Review by uekerman — APPROVED** [review]

**Pull request merged**